### ISSUE TITLE
Importing Configuration Migration - Unicode encoding

### ISSUE THREAD

**Issue by cseymr** [WaelHamze/xrm-ci-framework#269]
We're seeing a problem with Import Configuration.  Loosely speaking, when the imported text contains Emojis, they are showing in Unified Service Desk as a literal version of the Unicode value:

![image](https://user-images.githubusercontent.com/14763830/79855706-36d4d280-83c3-11ea-9e54-1557d8a017f5.png)

When we import the same file through the data config migration tool supplied by MS, we don't see this issue - the emojis are displayed correctly.

![image](https://user-images.githubusercontent.com/14763830/79855614-1c9af480-83c3-11ea-9783-8a476170bd4c.png)

In the source for the import, it looks like you are calling the MS Powershell directly, with no intervention, so we're wondering if the problem is with how MS are dealing with the data.xml file?

**Comment by WaelHamze** [comment]
@cseymr yes calling the powershell directly, the task is just a wrapper. Did you try to raise this with MS?
